test/runner/selenium/queue.js

```diff
@@ -3,33 +3,27 @@
 // and refills the queue when one promise resolves.
 
 import chalk from "chalk";
-import { getAvailableSessions } from "./browserstack/api.js";
-import { runWorker } from "./browserstack/workers.js";
-import { getBrowserString } from "./lib/getBrowserString.js";
-import { runSelenium } from "./selenium/runSelenium.js";
-import { runJSDOM } from "./jsdom.js";
+import { getBrowserString } from "../lib/getBrowserString.js";
+import { runSelenium } from "./runSelenium.js";
+import { runJSDOM } from "../jsdom.js";
 
-const queue = [];
 const promises = [];
+const queue = [];
 
 const SELENIUM_WAIT_TIME = 100;
-const BROWSERSTACK_WAIT_TIME = 5000;
-const WORKER_WAIT_TIME = 30000;
 
 // Limit concurrency to 8 by default in selenium
 // BrowserStack defaults to the max allowed by the plan
 // More than this will log MaxListenersExceededWarning
 const MAX_CONCURRENCY = 8;
 
-export function addRun( url, browser, options ) {
+export function addSeleniumRun( url, browser, options ) {
 	queue.push( { url, browser, options } );
 }
 
-export async function runFullQueue( {
-	browserstack,
-	concurrency: defaultConcurrency,
-	verbose
-} ) {
+// TODO: Possibly move this to selenium-only
+// and have a queue per browser
```

Comment:
Can you elaborate on this comment?

Reply:
Oops, this was written before I started this refactor. I've since done that refactor and can remove this comment.